src/app/features/ledger/components/ledger-inline-warnings.tsx

```diff
@@ -17,7 +22,7 @@ export function CommonLedgerDeviceInlineWarnings({
   if (outdatedLedgerAppWarning) {
     return (
       <WarningLabel fontSize="14px" textAlign="left">
-        Latest version of <em>Stacks app</em> required
+        Latest version of <em>{capitalize(chain)} app</em> required
```

Comment:
We could use CSS to capitalise this instead  `text-transform: capitalize`

Reply:
I think that would do all caps. This just capitalizes the first letter.

Reply:
Could use `::first-letter` selector to achieve the same. Not against this improvement, but used this as we already have the fn for js usecases too.

Reply:
There's a few variations of CSS transform: https://developer.mozilla.org/en-US/docs/Web/CSS/text-transform
```
/* Keyword values */
text-transform: none;
text-transform: capitalize;
text-transform: uppercase;
text-transform: lowercase;
```

`capitalize` just does the first letter. It's not a big deal to change now but we could include in in the design system typography. 

Then the browser will change it for us without needing a function to transform